### PR TITLE
SOC-1751 | Displaying posts title in discussions

### PR DESCRIPTION
**Pull request by Bloodmoon24**
https://wikia-inc.atlassian.net/browse/SOC-1751
@bkoval @kvas-damian  please CR.

### CODE REVIEW

File: front/styles/main/module/_discussion.scss [hunk 1]
```diff
@@ -71,6 +71,14 @@ body.discussions {
 		color: $discussion-content-color;
 		margin: 5px 0;
 		padding: .3rem 0;
+
+		h1, h2 {
+			margin: 0;
+		}
```
Comment (kvas-damian) [inline front/styles/main/module/_discussion.scss]: new line?

File: front/styles/main/module/_discussion.scss [hunk 2]
```diff
@@ -360,6 +368,11 @@ body.discussions {
 
 				.discussion-content {
 					line-height: $post-details-content-line-height;
+
+					h1 {
```
Comment (kvas-damian) [inline front/styles/main/module/_discussion.scss]: I think we change Rem base for bigger font on desktop already.

File: front/styles/main/module/_discussion.scss [hunk 3]
```diff
@@ -409,6 +422,11 @@ body.discussions {
 			.share {
 				display: none;
 			}
+
+			h1 {
```
Comment (kvas-damian) [inline front/styles/main/module/_discussion.scss]: What do you think about adding class name and styling using class?

**Comment by bkoval** [comment]
LGTM :haircut:

**Comment by kvas-damian** [comment]
LGTM

**Comment by kvas-damian** [comment]
LGTM

**Pull request merged**